plugins/ruby-foreman-foreman-tasks/debian/init.d

```diff
@@ -0,0 +1,274 @@
+#!/bin/bash
+#! /bin/sh
+### BEGIN INIT INFO
+# Provides:          foreman-tasks
+# Required-Start:    $remote_fs $syslog
+# Required-Stop:     $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Init script for foreman-tasks
+# Description:       Init script for foreman-tasks service used by foreman-tasks foreman plugin
+### END INIT INFO
+
+# Author: Marek Hulan <mhulan@redhat.com>
+
+PATH=/sbin:/usr/sbin:/bin:/usr/bin
+DESC="Foreman-tasks start dynflow executor"
+NAME=foreman-tasks
+DAEMON=/usr/bin/$NAME
+SCRIPTNAME=/etc/init.d/$NAME
+
+# Exit if the package is not installed
+[ -x "$DAEMON" ] || exit 0
+
+# Read configuration variable file if it is present
+[ -r /etc/default/$NAME ] && . /etc/default/$NAME
+
+# Load the VERBOSE setting and other rcS variables
+. /lib/init/vars.sh
+
+# Define LSB log_* functions.
+# Depend on lsb-base (>= 3.2-14) to ensure that this file is present
+# and status_of_proc is working.
+. /lib/lsb/init-functions
+
+RETVAL=0
+FOREMAN_USER=${FOREMAN_USER:-foreman}
+FOREMAN_HOME=${FOREMAN_HOME:-/usr/share/foreman}
+FOREMAN_DATA_DIR=${FOREMAN_DATA_DIR:-/var/lib/foreman}
+FOREMAN_ENV=${FOREMAN_ENV:-production}
+FOREMAN_TASK_PARAMS=${FOREMAN_TASK_PARAMS:--p foreman}
+FOREMAN_PID_DIR=$FOREMAN_HOME/tmp/pids
+FOREMAN_LOG_DIR=${FOREMAN_LOG_DIR:-/var/log/foreman}
+DAEMON="/usr/bin/foreman-tasks"
+PID_FILE=$FOREMAN_PID_DIR/dynflow_executor_monitor.pid
+export RAILS_ENV=$FOREMAN_ENV
+export FOREMAN_LOGGING=${FOREMAN_LOGGING:-warn}
+export FOREMAN_LOGGING_SQL=${FOREMAN_LOGGING_SQL:-warn}
+export RAILS_RELATIVE_URL_ROOT=$FOREMAN_PREFIX
+export BUNDLER_EXT_GROUPS
```

Comment:
Probably best to remove all references to BUNDLER_EXT, just to ensure it doesn't activate.